kivy/uix/screenmanager.py

```diff
@@ -964,6 +964,8 @@ def _get_screen_names(self):
     '''
 
     def __init__(self, **kwargs):
+        if 'transition' not in kwargs:
+            self.transition = SlideTransition()
```

Comment:
I'd perhaps choose to rewrite only the `kwargs` part and not put it directly into the instance because the `transition` is a Kivy property and those are evaluated from `kwargs` in the `super()` call to any `EventDispatcher` instance afaik. However when tested, the assignment triggers `on_transition` correctly, sooo.. still looks good to me :)

Reply:
Yes, that's what he suggested, but I felt like we didn't need that much indirection to simply set a variable, and I discussed with tshirtman in meatspace and we reached the same conclusion, so there you have it :-) if you still feel setting kwargs is a better idea, I can amend my PR of course!

Reply:
Well, it doesn't break the `on_*` event, so it should be ok.